starter-code/style.css

```diff
@@ -0,0 +1,144 @@
+.container1{
```

Comment:
Perhaps make the name of the selector "container*" more descriptive.
What does each class represent? If it selects a specific container, would an id perhaps be a better selector?